autopts/wid/sm.py

```diff
@@ -245,6 +253,22 @@ def hdl_wid_174(_: WIDParams):
     return True
 
 
+def hdl_wid_175(_: WIDParams):
+    """
+    Please confirm that IUT reports the Confirm Value Failed to the Upper Tester
+    """
+
+    stack = get_stack()
+    _, _, reason = stack.gap.gap_wait_for_pairing_fail(10)
+
+    # BT_SMP_ERR_CONFIRM_FAILED(0x04) is converted to BT_SECURITY_ERR_AUTH_FAIL(0x01)
+    # in zephyr host.
+    if reason == 0x01:
+        return True
```

Comment:
If this WID has specific behavior for Zephyr host, maybe it would be better to implement it in zephyr/sm_wid.py ?

Reply:
I'd suggest to define this on BTP level  (I'm fine if you want to map this one-to-one to zephyr enum)